config/src/test/java/org/polypheny/db/config/ConfigServerTest.java

```diff
@@ -17,26 +17,51 @@
 package org.polypheny.db.config;
 
 
+import com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import io.javalin.Javalin;
+import io.javalin.plugin.json.JavalinJackson;
 import java.util.Arrays;
 import java.util.Random;
 import java.util.Timer;
 import java.util.TimerTask;
-import org.polypheny.db.webui.ConfigServer;
+import org.polypheny.db.webui.ConfigService;
 
 
 public class ConfigServerTest {
```

Comment:
Is there still a purpose for this class?

Reply:
No, removed.